server/sonar-server/src/main/java/org/sonar/server/issue/ws/SearchAction.java

```diff
@@ -140,7 +140,7 @@ public void define(WebService.NewController controller) {
     action.addSortParams(IssueQuery.SORTS, null, true);
     action.createParam(ADDITIONAL_FIELDS)
       .setSince("5.2")
-      .setDescription("Comma-separated list of the optional fields to be returned in response.")
+      .setDescription("Comma-separated list of the optional fields to be returned in response. Action plans are dropped in 5.5, it is not returned in the response.")
```

Comment:
Description of the WS should now mention that action plan related fields are deprecated